src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4022,17 +4022,23 @@ public override IOperation VisitCaughtException(ICaughtExceptionOperation operat
 
         public override IOperation VisitInvocation(IInvocationOperation operation, int? captureIdForResult)
         {
-            if (operation.Instance != null)
+            IOperation instance = operation.TargetMethod.IsStatic ? null : operation.Instance;
+            (IOperation visitedInstance, ImmutableArray<IArgumentOperation> visitedArguments) = VisitInstanceWithArguments(instance, operation.Arguments);
+            return new InvocationExpression(operation.TargetMethod, visitedInstance, operation.IsVirtual, visitedArguments, semanticModel: null, operation.Syntax,
+                                            operation.Type, operation.ConstantValue, IsImplicit(operation));
+        }
+
+        private (IOperation, ImmutableArray<IArgumentOperation>) VisitInstanceWithArguments(IOperation instance, ImmutableArray<IArgumentOperation> arguments)
```

Comment:
Consider adding names to the tuple elements. My personal rule of thumb (not sure we have an established policy here). If it's on a non-local function then generally best to add names.